app/models/queen.rb

```diff
@@ -1,2 +1,8 @@
 class Queen < ChessPiece
+  def display
+    if color == "black"
+      return "&#9819;"
+    else "&#9813"
```

Comment:
Should follow the same structure with a return "&#9813" being under the else statement.